drivers/dma/dma_stm32.c

```diff
@@ -271,7 +271,9 @@ DMA_STM32_EXPORT_API int dma_stm32_configure(const struct device *dev,
 		return -EBUSY;
 	}
 
-	stm32_dma_disable_stream(dma, id);
+	while (stm32_dma_disable_stream(dma, id) != 0) {
```

Comment:
One last question. Due to software or hardware bugs, this bit may not be able to set to the desired state. Using an endless while may cause hardware stuck here, especially when we are in the starting of the kernel. I think here we should wait 1~5 seconds, report the error and return. What do you think?